src/components/ui/Box/Box.scss

```diff
@@ -5,4 +5,9 @@
   border-style: solid;
   border-width: px-to-rem(1px);
   padding: pad(large) pad(small);
+  transition: 0.4s;
+
+  &:hover {
+    box-shadow: 1px 1px 30px rgba(51, 51, 51, 0.3);
+  }
```

Comment:
naçizane fikrimdir :) features box'lar biraz daha canlanırsa daha iyi görüneceğini düşündüm.

Reply:
![image](https://user-images.githubusercontent.com/1509413/39471898-18289926-4d46-11e8-9953-a236505f8f54.png)

Güzel bir dinamizm kattığı kesin. "new-jargon" etiketi altında olmasaydı keşke :D out of context diyor ama yine de beğendiğimi de belirtmeden edemiyorum.

Reply:
Gorsel acidan guzel ama hover'da boyle bir efekt yapinca insan o cursor'un pointer olmasini dolayisiyla da hover edilen seyin tiklanabilir bir sey olmasini umuyor, kullaniciyi yaniltiyor gibi. Siz ne dusunuyorsunuz @gokaygurcan @erkansivas35?

Reply:
ne yalan söyliyim ilk yaptığım şey tıklamak oldu benim de denerken :)